Automattic/jetpack
# JS Components: Convert icons to TypeScript

#### Changes proposed in this Pull Request:
* This PR converts all the icons to TypeScript.
* It also converts the icons' stories to TS

#### Other information:


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Checkout the PR
* Run `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Confirm that all the stories of all the icons work as expected
* Check the consumers of icons and related components and confirm that they work as expected
* Check for any regression

<img width="1432" alt="Screenshot 2022-07-27 at 10 52 10 AM" src="https://user-images.githubusercontent.com/18226415/181167551-5ab2c88e-2cc6-4588-ac74-fec54e1600ea.png">



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202276314263999
